src/dotty/tools/dotc/repl/CombinedClassLoader.scala

```diff
@@ -0,0 +1,37 @@
+package dotty.tools
+package dotc
+package repl
+
+import scala.util.Try
+
+class CombinedClassLoader(loaders: List[ClassLoader]) extends ClassLoader {
```

Comment:
They just set it as the class loader - but that doesn't work for us. So I decided that it would be a good idea to merge the ClassLoaders and look through both of them when searching for classes.

But I don't like this solution - if you have a better one - I'm all ears.